plugins/modules/sonic_vlans.py

```diff
@@ -96,6 +101,7 @@
 #sonic#
 #
 #sonic# show interface Vlan 10
+Description: Internal
```

Comment:
It looks like this line should have a preceding "#" (for consistency with the adjacent lines).